docs/user-guide/global-configurations/authorization/sso/microsoft.md

```diff
@@ -0,0 +1,66 @@
+# Microsoft
+
+## Sample Configuration
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/sso-login-service/microsoft.jpg)
+
+---
+
+## Values You Would Require at SSO Provider
+
+Devtron provides a sample configuration out of the box. There are some values that you need to either get from your SSO provider or give to your SSO provider.
+
+### Values to Fetch
+
+* clientID
+
+* tenantID (required only if you want to use Azure AD for auto-assigning permissions)
+
+    ![Fetching Client ID and Tenant ID](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/sso-login-service/secret/ms-id-secret1.jpg)
+
+* clientSecret
+
+    ![Fetching Secret](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/sso-login-service/secret/ms-id-secret2.jpg)
+
+### Values to Provide
+
+* redirectURI (provided in SSO Login Services by Devtron)
+
+    ![Pasting Redirect URI](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/sso-login-service/redirect/ms-redirect.jpg)
+
+---
+
+## Reference
+
+[View Microsoft Documentation](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app)
+
+---
+
+## Auto-assign Permissions [![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/elements/EnterpriseTag.svg)](https://devtron.ai/pricing)
+
+{% hint style="info" %}
+Only a super-admin can enable this feature. Make sure to add tenantID in the SSO configuration field without fail.
```

Comment:
Only superadmin can configure anything related to SSO, not just this. Also who's superadmin if no SSO is configured? :)